src/client/public/try.it.ts

```diff
@@ -0,0 +1,38 @@
+import * as $ from 'jquery';
+import '../assets/styles/extras.scss';
+
+interface InitializationParams {
+    origin: string;
+    runnerSnippetUrl: string;
+    wacUrl: string;
+}
+
+(() => {
+    (window as any).in_try_it_mode = true;
+    (window as any).initializeTryIt = initializeTryIt;
+    let snippetId;
+
+    function initializeTryIt(params: InitializationParams): void {
+        $(document).ready(() => {
+            window.addEventListener('message', receiveMessage, false);
+            let url = params.wacUrl;
+            let session = new (OfficeExtension as any).EmbeddedSession(url, { id: 'embed-frame', sessionKey: '1', container: document.getElementById('panel-bottom') });
```

Comment:
I don't think you need to specify sessionKey, office.js should generate this for you.